packages/firebase_vertexai/firebase_vertexai/lib/firebase_vertexai.dart

```diff
@@ -29,54 +32,53 @@ export 'src/api.dart'
         PromptFeedback,
         SafetyRating,
         SafetySetting,
-        // TODO(cynthiajiang) remove in next breaking change.
-        TaskType,
-        UsageMetadata;
-export 'src/base_model.dart'
-    show GenerativeModel, ImagenModel, LiveGenerativeModel;
-export 'src/chat.dart' show ChatSession, StartChatExtension;
-export 'src/content.dart'
-    show
+        UsageMetadata,
```

Comment:
I was planning to remove this field, since there's no use case for developer to instantiate the UsageMetadata. Do you think it's a good time for this change?

Reply:
We can drop the export of TaskType since I think that's unused.

Do we really want to remove this type? It's still user visible right? We can consider removing the public constructor, but if we expect user code to read from this class I think we do want to export it.

Reply:
Sounds good, let's remove the public constructor in a follow up PR.